volpe/compile.py

```diff
@@ -84,8 +102,17 @@ def compile_and_run(llvm_ir, result_type, more_verbose=False, show_time=False, c
     print("main() =", repr(res))
     
     if show_time:
-        time_taken = end_time - start_time
-        if time_taken > 100000:
-            print(f"time = {time_taken/1E9}s")
+        if count > 1:
+            print(f"ran the code {count} times")
+            nanoseconds = MEASUREMENT_TIME / count * (1E9 / ticks_in_sec) 
```

Comment:
why is this `MEASUREMENT_TIME` instead of `end_time - start_time`?

Reply:
I forgot to update that. It works fine as it is, but `end_time - start_time` might be slightly more accurate. At one point during the improvement I wasn't tracking `end_time` so I used `MEASUREMENT_TIME` instead; I will change it.